### ISSUE TITLE
Minion skills/gems in offensive? 

### ISSUE THREAD

**Issue by AngryRoleplayer** [EmmittJ/PoESkillTree#450]
Hi there! Can't see what i am doing wrong. I have zombies/srs/spectres but can't find where i can check their damage in the offensive ?? No support for minions? 
![snag-0024](https://cloud.githubusercontent.com/assets/22470478/23847598/338d95ca-07e4-11e7-8cc3-10385b438191.jpg)

**Comment by brather1ng** [comment]
No, minions are not supported. They require data outside of skill gems that we don't have.

**Comment by EmmittJ** [comment]
Ah, I answered this in the forums as well. Missed this. We pretty much
don't have any data on minions since 2.0, iirc. I would say that we won't
really ever be able to fully support them.

**Comment by AngryRoleplayer** [comment]
Got it, thanx!

**Comment by AngryRoleplayer** [comment]
Minion data has been released by GGG officially. Do you plan to add that info to the program? Path of Building already did it

**Comment by EmmittJ** [comment]
We are slowly getting there; we still have a long way to go before we are
close to the level of accuracy of Path of Building. Realistically minions
will be a later addition
